src/coverlet.collector/DataCollection/CoverletCoverageCollector.cs

```diff
@@ -132,7 +131,7 @@ private void OnSessionStart(object sender, SessionStartEventArgs sessionStartEve
                 _serviceProvider = _serviceCollectionFactory(_eqtTrace, _logger, coverletSettings.TestModule).BuildServiceProvider();
 
                 // Get coverage and attachment managers
-                _coverageManager = new CoverageManager(coverletSettings, _eqtTrace, _logger, _coverageWrapper,
+                _coverageManager = new CoverageManager(coverletSettings, _eqtTrace, _serviceProvider.GetRequiredService<ILogger>(), _coverageWrapper,
```

Comment:
Keep master version, we pass also `_eqTrace` and it's ok, it's the same object if we have access we can use it.

Reply:
Done.